packages/@lwc/features/src/babel-plugin/index.js

```diff
@@ -1,6 +1,10 @@
 const template = require('@babel/template');
 
-const buildRequire = template.expression('global.LWC_config.features.FEATURE_FLAG');
+const buildRequire = template.expression(`
+    global.LWC_config &&
```

Comment:
I think we can guarantee `global.LWC_config.features` to always be present by setting up that during initialization of the config code.